tests/unit/reports/single_table/_properties/test_synthesis.py

```diff
@@ -0,0 +1,86 @@
+from unittest.mock import Mock, call, patch
+
+import pandas as pd
+
+from sdmetrics.reports.single_table._properties.synthesis import Synthesis
+
+
+class TestSynthesis:
+
+    @patch('sdmetrics.reports.single_table._properties.synthesis.'
+           'NewRowSynthesis.compute_breakdown')
+    def test__generate_details(self, newrowsynthesis_mock):
```

Comment:
can we add a unit test for the error case as well

Reply:
Yes I was doing it hahaha. Done in [a310a75](https://github.com/sdv-dev/SDMetrics/pull/398/commits/a310a7541be76a3cc5fac3b80250bbc1b7f8c665)